src/components/slider/slider.e2e.ts

```diff
@@ -673,4 +673,49 @@ describe("calcite-slider", () => {
   it("is form-associated", () => formAssociated("calcite-slider", { testValue: 5 }));
 
   it("is form-associated with range", () => formAssociated("calcite-slider", { testValue: [5, 10] }));
+
+  it("groupSeparator: displays numbers as comma separated when prop is true and does not when false", async () => {
+    const page = await newE2EPage({
+      html: `<calcite-slider 
+                min="1000" 
+                max="10000" 
+                min-value="2500" 
+                max-value="5000" 
+                step="1000" 
+                ticks="1000" 
+                label-handles 
+                label-ticks 
+                style="width:${sliderWidthFor1To1PixelValueTrack}">
+            </calcite-slider>`
+    });
+    await page.waitForChanges();
+    const element = await page.find("calcite-slider");
+
+    const displayedValuesArray = async (): Promise<string[]> => {
+      await page.waitForChanges();
+
+      const labelMinVal = (await element.shadowRoot.querySelector("span.handle__label--minValue")) as HTMLElement;
+      const labelVal = (await element.shadowRoot.querySelector("span.handle__label--value")) as HTMLElement;
+
+      const tickMin = (await element.shadowRoot.querySelector("span.tick__label--min")) as HTMLElement;
+      const tickMax = (await element.shadowRoot.querySelector("span.tick__label--max")) as HTMLElement;
+
+      return [labelMinVal.innerText, labelVal.innerText, tickMin.innerText, tickMax.innerText];
+    };
+    await page.exposeFunction("displayedValuesArray", displayedValuesArray);
+
+    const noSeparator = await page.$eval("calcite-slider", async (): Promise<string[]> => {
+      return await displayedValuesArray();
+    });
+
+    element.setProperty("groupSeparator", true);
+    await page.waitForChanges();
+
+    const withSeparator = await page.$eval("calcite-slider", async (): Promise<string[]> => {
+      return await displayedValuesArray();
+    });
+
+    expect([...noSeparator]).toEqual(["2500", "5000", "1000", "10000"]);
+    expect([...withSeparator]).toEqual(["2,500", "5,000", "1,000", "10,000"]);
+  });
```

Comment:
Great test! Can you update it to cover additional locales? Something like https://github.com/Esri/calcite-components/blob/adcbb55b3ab3ab9fca0921b4bbbea1e4bf92907b/src/components/input/input.e2e.ts#L1057 would be good.